multidim-interop/impl/go/v0.27/Makefile

```diff
@@ -12,7 +12,7 @@ go-libp2p-${version}: go-libp2p-${version}.zip
 	unzip -o go-libp2p-${version}.zip
 
 go-libp2p-${version}.zip:
-	wget -O $@ "https://github.com/libp2p/go-libp2p/archive/v${version}.zip"
+	wget --secure-protocol=TLSv1_2 -O $@ "https://github.com/libp2p/go-libp2p/archive/v${version}.zip"
```

Comment:
Why is this needed? Just use curl?

Reply:
Do you have a strong preference for curl vs wget?

the reason this is needed is explained in the linked GitHub discussion from an earlier comment: https://github.com/orgs/community/discussions/65227

Reply:
The link says that curl doesn't need that hack, which makes curl look like the better choise. Also, I know the curl author and I trust him to do the right thing.

Reply:
Fine by me